Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -2647,6 +2638,10 @@ class PaymentSheetDeferredServerSideUITests: PaymentSheetUITestCase {
 
         try! fillCardData(app)
 
+        var saveThisCardToggle = app.switches["Save this card for future Example, Inc. payments"]
+        XCTAssertFalse(saveThisCardToggle.isSelected)
+        saveThisCardToggle.tap()
+
```

Comment:
Nit: seems like we should add  (for multiple of these)
```
        saveThisCardToggle.tap()
+       XCTAssertTrue(saveThisCardToggle.isSelected)
```

Reply:
I think the test enforces this (it checks if the card is saved!) but I can